cmd/resolutioncli/main.go

```diff
@@ -22,8 +22,7 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/operator-framework/deppy/pkg/deppy/solver"
```

Comment:
nit: likely good to use an import sorting linter to remove these diffs in the future